src/main/java/nl/ultimateapps/TechItEasy/controllers/TelevisionController.java

```diff
@@ -1,13 +1,67 @@
 package nl.ultimateapps.TechItEasy.controllers;
 
-
+import nl.ultimateapps.TechItEasy.exceptions.RecordNotFoundException;
+import nl.ultimateapps.TechItEasy.service.Television;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
 
 @RestController
```

Comment:
Je kunt hier nog @RequestMapping("/televisions") toevoegen dan kun je "/televisions" weglaten in alle andere endpoints die je aanmaakt in deze controller